test/windows/UnitTests.cpp

```diff
@@ -842,6 +842,17 @@ class UnitTests
                 L"Invalid command line argument: --invalid\nPlease use 'wslinfo --help' to get a list of supported "
                 L"arguments.\n");
         }
+
+        {
+            // Get the VM ID from the distro and validate that it not null.
+            auto [vmId, vmIdErr] = LxsstuLaunchWslAndCaptureOutput(L"env | grep 'WSL2_VM_ID' | awk -F= '{print $2}'");

```

Comment:
This test case is failing when the tests run in WSL1 mode. Let's wrap in in ` if (LxsstuVmMode())`  like we do for `--networking-mode` above

Reply:
I'll get that sorted now! Thanks.

Reply:
Done!